lib/bootstrap.rb

```diff
@@ -0,0 +1,118 @@
+module LogStash
+  module Environment
```

Comment:
Why are now bootstrap and environment mix ? as for the main responsibilities of both, I see them in separate files for clarity. I see times where I might need to load the env without having to bootstrap.